tests/general/style.py

```diff
@@ -8,7 +8,6 @@ def test_style():
     E501 - Line too long (82 > 79 characters)
     W504 - Line break occurred after a binary operator
     """
-
```

Comment:
btw, this entire test can be deleted, as it's deprecated in favor of flake8

Reply:
makes sense, I deleted it